safeutils/message_hash.go

```diff
@@ -0,0 +1,72 @@
+package safeutils
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+var SAFE_MESSAGE_TYPEHASH = common.HexToHash("0x60b3cbf8b4a223d68d641b3b6ddf9a298e7f33710cf3d3a9d1146b5a6150fbca")
```

Comment:
Could we add a comment to describe what it hashes?

`// keccak256("SafeMessage(bytes message)");`

Reply:
Addressed in https://github.com/trishtzy/gosafe/pull/2/commits/310b8416bda5cca3848c7a075cf08f873e67a923